### DISCUSSION TITLE
Implement ThumbHash

### DISCUSSION THREAD

**Discussion by jagu-sayan** [directus/directus#19475] [Feature Requests]
### Summary

Provide a base64 image placeholder with [ThumbHash](https://github.com/evanw/thumbhash)

### Motivation

To create a seamless loading experience for the user and reducing the perceived loading time.

### Detailed Design

Like described in this article : https://www.datocms.com/blog/introducing-thumbhash-the-compact-placeholder-with-alpha-support

### Drawbacks

- can be done on client side

### Alternatives

- [blurhash](https://blurha.sh/)
- https://plaiceholder.co/docs

### Adoption Strategy

Small improvement with no impact on current Directus ecosystem.

**Comment by DanielBiegler** [comment]
The comparison section on the site is quite nice to get an overview! See: https://evanw.github.io/thumbhash/

This got requested recently too over at #19443 and has some upvotes there already. Lets move this discussion over there. :)
